pages/peterportal/getting-started.mdx

```diff
@@ -0,0 +1,76 @@
+import { Steps } from "nextra/components";
+
+# Getting Started
+
+This document will guide you through setting up the project on your local machine, from cloning the repository to running the project locally. Whether you're a committee member or an open-source contributor, you'll find all the necessary steps to get started below.
+
+## Setup
+
+<Steps>
+
+### Cloning the Repository
+
+#### For Committee Members
+
+1. Clone the repository using the following command:
+   `git clone https://github.com/icssc/peterportal-client`
+2. Navigate into the cloned repository:
+   `cd peterportal-client`
+
+#### For Open Source Contributors
+
+1. Fork the project on GitHub by clicking the fork button in the top right, above the "About" section.
+2. Clone your forked repository to your local machine using the following command (replace `your username` with your GitHub username):
+   `git clone https://github.com/yourusername/peterportal-client`
+3. Navigate into the cloned repository:
+   `cd peterportal-client`
+
+### Installing Dependencies
+
+1. Ensure you have Node.js version 18 or 20 LTS installed. If not, consider using nvm for version management.
+2. Install all node dependencies by running:
+   `npm install`
+
+### Setting up Environment Variables
+
+1. Create a `.env` file in the `api` directory with the necessary environment variables as provided by the project lead or documented below for open-source contributors.
+
+For Open Source Contributors, your `.env` file should include:
+
+`PUBLIC_API_URL=https://api-next.peterportal.org/v1/rest/`
+
+`PUBLIC_API_GRAPHQL_URL=https://api-next.peterportal.org/v1/graphql`
+
+`PORT=8080`
```

Comment:
Actually, just updated the repo to have an example env file under `api/.env.example`. I figured this would make it easier for open source contributors. Let's adjust the instructions to tell them to rename that file to `.env`.